portfolio/src/main/webapp/index.html

```diff
@@ -35,33 +32,36 @@
     <div id="navbar" class="navbar-collapse">
         <ul class="nav navbar-nav">
             <li><a href="index.html">Home</a></li>
-            <li><a href="resume.pdf">Resume</a></li>
+            <li><a href="images/resume.pdf">Resume</a></li>
```

Comment:
I would suggest to have the resume in a different folder than images/ (maybe something like documents/ ? or since it's a portfolio, you can more specific and call it resume/). It will help keeping things clean as the project grow in size.